infra/Pos/Communication/Relay/Logic.hs

```diff
@@ -362,10 +359,11 @@ invReqDataFlowNeighbors
     , Bi tag, Bi id
     , Bi (InvOrData tag id contents)
     , Bi (ReqMsg id tag))
-    => Text -> SendActions m -> tag -> id -> contents -> m ()
-invReqDataFlowNeighbors what sendActions tag id dt = handleAll handleE $
-    reifyMsgLimit (Proxy @(ReqMsg id tag)) $ \lim ->
-        converseToNeighbors sendActions (invReqDataFlowDo what tag id dt lim)
+    => m (Set NodeId) -> Text -> SendActions m -> tag -> id -> contents -> m ()
+invReqDataFlowNeighbors getPeers what sendActions tag id dt = handleAll handleE $
+    reifyMsgLimit (Proxy @(ReqMsg id tag)) $ \lim -> do
+        peers <- getPeers
+        converseToNeighbors peers sendActions (invReqDataFlowDo what tag id dt lim)
```

Comment:
Seems like pattern `getPeers >>= \peers -> converseToNeighbors peers sendActions patak` is pretty common. Why not create a wrapper for it?